project/profiles/capacity4more/themes/c4m/kapablo/build/src/stylesheets/components/_activity_stream.scss

```diff
@@ -17,12 +28,23 @@
   h2 {
     margin-top: 25px;
   }
-  
+
   .message-summary {
     padding: 0;
   }
 }
 
+.pane-og-dashboard {
```

Comment:
@nirgn975 
if I remember correct, we have this flag also on the group dashboard.
please make sure it doesn't effect this one as well...